retriever/bin/retriever.js

```diff
@@ -119,6 +116,7 @@ export default {
       )
       const response = new Response(originResponse.body, originResponse)
       setContentSecurityPolicy(response)
+      response.headers.set('X-Dataset-ID', setId)
```

Comment:
Has FilOz definitely decided on this terminology? I see a risk of making things more confusing if we integrate this new name too early (and nothing else is using it yet)

Reply:
See [d229586](https://github.com/filcdn/worker/pull/163/commits/d229586d5b8776352863905503b2230c2e00a4a4)


I decided to use `X-Proof-Set-ID` because "proofset" is not a word.


<img width="820" height="413" alt="Screenshot 2025-07-16 at 11 41 53" src="https://github.com/user-attachments/assets/72cd4123-ed35-4303-8a7e-25abbe01362d" />